popup/popup.html

```diff
@@ -9,24 +9,28 @@
 
 <body>
   <p>
-    <center>Remind me to drink water for every</center>
```

Comment:
center tag is not supported in HTML 5. I think you can add a `text-align` style in css.

Reply:
nvm! we can add it later.